metaflow/plugins/aws/batch/batch.py

```diff
@@ -113,10 +113,14 @@ def create_job(
         image,
         queue,
         iam_role=None,
+        execution_role=None,
         cpu=None,
         gpu=None,
         memory=None,
         run_time_limit=None,
+        shared_memory_size=None,
```

Comment:
nit: you could rename `shared_memory_size` to just `shared_memory`, so it is consistent with `memory` and other attributes

Reply:
Done!